workflows/check-release-notes.yml

```diff
@@ -0,0 +1,39 @@
+name: Check release notes in the PR description
+
+on:
+  workflow_call:
+    inputs:
+      description:
+        required: true
+        type: string
+      placeholder:
+        required: true
+        type: string
+
+jobs:
+  release-notes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if there are any release release notes and placeholder
+        env:
+          PR_DESCRIPTION: ${{ inputs.description }}
+          NOTES_PLACEHOLDER: ${{ inputs.placeholder }}
+        shell: python
+        run: |
+          import os
+
+          RELEASE_NOTES_TAG = "RELEASE NOTES"
```

Comment:
why not directly including the ` BEGIN`?

Reply:
taken it from the `prepare-release`, where we have just this part in a constant (that could be changed, if need be) and `BEGIN` is included below…